build.gradle

```diff
@@ -532,6 +489,21 @@ allprojects {
   }
 }
 
+
+tasks.register("precommit") {
+  dependsOn gradle.includedBuild('build-tools').task(':precommit')
+  dependsOn gradle.includedBuild('build-tools-internal').task(':precommit')
+}
+
+tasks.named("checkPart1").configure {
+  dependsOn gradle.includedBuild('build-tools').task(':checkPart1')
```

Comment:
I don't think we need to create a "checkPart1" task on these projects. The point here is to distribute these checks to different CI jobs. We can just depend on the included builds' `check` task here.

Reply:
I simplified this as suggested